### PR TITLE
vpc bandwidth resource

### PR DESCRIPTION
**Pull request by Polina-Gubina**

### CODE REVIEW

**Comment by vladimirvshivkov** [comment]
Could you attach test results, please?

File: otcextensions/sdk/vpc/v1/_proxy.py [hunk 1]
```diff
@@ -21,6 +22,65 @@
...
+        """Assign bandwidth
+
+        :param dict attrs: Keyword arguments which will be used to assign
+            a :class:`~otcextensions.sdk.vpc.v1.bandwidth.Bandwidth`
+        """
+        project_id = self.get_project_id()
+        return self._create(_bandwidth.Bandwidth, project_id=project_id,
+                            **attrs)
+
+    def add_eip_to_bandwidth(self, bandwidth, publicip_info):
```
Comment (anton-sidelnikov) [inline otcextensions/sdk/vpc/v1/_proxy.py]: please add more detailed info in doc about publicip_info structure

File: otcextensions/sdk/vpc/v1/bandwidth.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    #: Specifies the enterprise project ID.
+    enterprise_project_id = resource.Body('enterprise_project_id', type=str)
+    #: Specifies the bandwidth status.
+    status = resource.Body('status', type=str)
+    #: Specifies the time (UTC) when the bandwidth is created.
+    created_at = resource.Body('created_at', type=str)
+    #: Specifies the time (UTC) when the bandwidth is updated.
+    updated_at = resource.Body('updated_at', type=str)
+
+    def add_eip_to_bandwidth(self, session, publicip_info, project_id):
```
Comment (anton-sidelnikov) [inline otcextensions/sdk/vpc/v1/bandwidth.py:71]: maybe is a good idea to make publicip_info of PublicIPInfo type

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: otcextensions/sdk/vpc/v1/_proxy.py (hunk 1 above)
Comment (gtema) [reply otcextensions/sdk/vpc/v1/_proxy.py]: I would say include this into docs as such

**Review by anton-sidelnikov — DISMISSED** [review]

**Review by anton-sidelnikov — APPROVED** [review]

**Pull request merged**